docs/guides/Screen-Navigation-Prop.md

```diff
@@ -116,6 +116,26 @@ class HomeScreen extends React.Component {
 }
 ```
 
+*Notes regarding key on go back from*
```

Comment:
I think this should be reworded for clarity.

Maybe something like:

Going Back from a Specific Screen

I'm not a huge fan of naming sections of documentation as "Notes on ..." because that title does not provide value for someone trying to skim through the documentation.